airbyte-integrations/connectors/source-stripe/unit_tests/conftest.py

```diff
@@ -34,46 +34,13 @@ def incremental_args_fixture(stream_args):
     return {"lookback_window_days": 14, **stream_args}
 
 
-@pytest.fixture(name="invoices")
-def invoices_fixture(stream_args):
-    def mocker(args=stream_args):
-        return IncrementalStripeStream(
-            name="invoices",
-            path="invoices",
-            use_cache=False,
-            event_types=[
-                "invoice.created",
-                "invoice.finalization_failed",
-                "invoice.finalized",
-                "invoice.marked_uncollectible",
-                "invoice.paid",
-                "invoice.payment_action_required",
-                "invoice.payment_failed",
-                "invoice.payment_succeeded",
-                "invoice.sent",
-                "invoice.upcoming",
-                "invoice.updated",
-                "invoice.voided",
-            ],
-            **args,
-        )
-
-    return mocker
-
-
-@pytest.fixture(name="invoice_line_items")
-def invoice_line_items_fixture(invoices, stream_args):
-    parent_stream = invoices()
-
-    def mocker(args=stream_args, parent_stream=parent_stream):
-        return StripeLazySubStream(
-            name="invoice_line_items",
-            path=lambda self, *args, stream_slice, **kwargs: f"invoices/{stream_slice[self.parent_id]}/lines",
-            parent=parent_stream,
-            parent_id="invoice_id",
-            sub_items_attr="lines",
-            add_parent_id=True,
-            **args,
-        )
+@pytest.fixture()
+def stream_by_name(config):
+    def mocker(stream_name, source_config=config):
+        source = SourceStripe()
```

Comment:
I really like that because it uses the streams as configured by the source so not only it tests the streams but also how the source instantiate them